x-pack/plugins/security_solution/public/alerts/components/alerts_table/translations.ts

```diff
@@ -135,3 +159,10 @@ export const OPENED_ALERT_FAILED_TOAST = i18n.translate(
     defaultMessage: 'Failed to open alert(s)',
   }
 );
+
+export const IN_PROGRESS_ALERT_FAILED_TOAST = i18n.translate(
+  'xpack.securitySolution.detectionEngine.alerts.inProgressAlertFailedToastMessage',
+  {
+    defaultMessage: 'Failed to mark alert(s) as in progress',
```

Comment:
You can keep the `(s)` if you really want to but there are plural i18n like the one right above this `IN_PROGRESS_ALERT_SUCCESS_TOAST` where if you want you can optionally determine if it is a single alert vs a plural alert and then change the next accordingly.

Reply:
right now the action in which this is used isn't set up to pass the amount of alerts successfully/unsuccessfully updated in the `catch` part of the try catch so we're leaving it like this for all the error toasts that deal with updating until that change gets made